components/41-organisms/footer/footer.twig

```diff
@@ -22,7 +22,7 @@
               'icon_items': [
                 '<i class="icon-phone" aria-hidden="true"></i><a href="tel:' ~ gentinfo.phone.machine ~ '" alt="Gentinfo tel link">' ~ gentinfo.phone.label ~ '</a>',
                 '<i class="icon-envelope" aria-hidden="true"></i><a href="mailto:' ~ gentinfo.email ~ '" alt="Gentinfo mail link">' ~ gentinfo.email ~ '</a>',
-                '<i class="icon-document" aria-hidden="true"></i><a href="#">' ~ gentinfo.form ~ '</a>'
+                '<i class="icon-document" aria-hidden="true"></i><a href="https://stad.gent/contact">' ~ gentinfo.form ~ '</a>'
```

Comment:
geen alt?

Reply:
Verdorie niet gezien dat daar ook verkeerdelijk alts staan...
Ik haal ze snel overal weg. Dat is geen geldig attribuut op een anchor.